packages/config/src/configDefaults.ts

```diff
@@ -286,6 +286,7 @@ export const configProps = ({
 
         const options = configObject.network_config;
         options.verboseRpc = configObject.verboseRpc;
+        options.events = configObject.events;
```

Comment:
Why'd this need to be added here now?

Reply:
Without this line the `events` field wasn't set on the `config` object at the time that the `wrap` function is called in `@truffle/provider`. When that's the case, the `preHook` and `postHook` don't fire off their respective `rpc:*` events.

Reply:
[block where event is emitted](https://github.com/trufflesuite/truffle/blob/12c76a0ec9647c94400101e6dd1fc2249aeb8806/packages/provider/wrapper.js#L87-L92)